openmls/src/treesync/node/leaf_node/lifetime.rs

```diff
@@ -57,11 +61,24 @@ impl Lifetime {
 
     /// Returns true if this lifetime is valid.
     pub(crate) fn is_valid(&self) -> bool {
-        let now = SystemTime::now()
+        match SystemTime::now()
             .duration_since(UNIX_EPOCH)
-            .expect("SystemTime before UNIX EPOCH!")
-            .as_secs();
-        self.not_before < now && now < self.not_after
+            .map(|duration| duration.as_secs())
+        {
+            Ok(elapsed) => self.not_before < elapsed && elapsed < self.not_after,
+            Err(_) => {
+                log::error!("SystemTime before UNIX EPOCH.");
+                false
+            }
+        }
+    }
+
+    /// ValSem(openmls/annotations#32):
```

Comment:
Not sure to what issue to link this. After I gaines a bit of experience with the annotation tool, I think it might be best to stick to the current `ValSem` approach -- maybe in a way that can be reflected using GitHub issues? Sometimes, multiple annotations refer to the same validation procedure but are phrased differently in the standard.

Reply:
We should probably have a general discussion on how to enumerate/name validation steps. The annotation tool wasn't around when the ValSem notation was introduced. I'd say leave it as it is for now.